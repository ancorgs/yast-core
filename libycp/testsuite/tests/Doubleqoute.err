Parsed:
----------------------------------------------------------------------
``((1 + 2))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
``(name)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
``{
  return (3 + 4);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  return 7;
}
----------------------------------------------------------------------
