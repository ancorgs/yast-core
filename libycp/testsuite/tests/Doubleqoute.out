((1 + 2))
(name)
({
  return (3 + 4);
}
)
(7)
